test/integration/api/services_controller_test.rb

```diff
@@ -25,28 +25,6 @@ class SettingsTest < self
       assert_response :success
     end
 
-    test 'settings with finance allowed' do
-      Account.any_instance.stubs(:provider_can_use?).returns(true)
-      Account.any_instance.stubs(:provider_can_use?).with(:api_as_product).returns(false | true)
-      rolling_update(:api_as_product, enabled: false)
-
-      provider.settings.finance.allow
-
-      get settings_admin_service_path(service)
-
-      assert_select "input[name='service[buyer_plan_change_permission]'][value=credit_card]"
-      assert_select "input[name='service[buyer_plan_change_permission]'][value=request_credit_card]"
-    end
-
-    test 'settings with finance denied' do
-      provider.settings.finance.deny
-
-      get settings_admin_service_path(service)
-
-      assert_select "input[name='service[buyer_plan_change_permission]'][value=credit_card]", false
-      assert_select "input[name='service[buyer_plan_change_permission]'][value=request_credit_card]", false
-    end
```

Comment:
Shouldn't it be the same for APIAP enabled?

Reply:
I though so, but something in the UI must have changed and those input don't exist anymore. I'll try again.

Reply:
Verified the tests will only pass without APIAP. I don't know where's finance now but it is not in the settings page, so the tests are not longer valid.